static/js/biomedical/disease/chart.ts

```diff
@@ -247,3 +259,82 @@ export function drawDiseaseSymptomAssociationChart(
       (d) => `Symptom: ${d.name}<br>Odds Ratio Association: ${d.oddsRatio}`
     );
 }
+/**
+ * Draws the disease ontology hierarchy chart for the disease of interest
+ * @param id the div id where the chart is rendered on the page
+ * @param data the disease data passed into the function
+ */
+export function drawDiseaseOntologyHierarchy(
+  id: string,
+  data: DiseaseParentTree[]
+): void {
+  const height = GRAPH_HEIGHT - MARGIN.top - MARGIN.bottom;
+  const width = GRAPH_WIDTH - MARGIN.left;
+  // tree root
+  const root = d3.hierarchy(data).sort((a, b) => b.height - a.height);
+  // tree layout
+  const treeLayout = d3.tree().size([600, 120]);
+  // calling the tree layout function for the rearrangement of root
+  treeLayout(root);
+
+  const svg = d3
+    .select(`#${id}`)
+    .append("svg")
+    .attr("width", width)
+    .attr("height", height)
+    .append("g")
+    .attr("transform", "translate(" + MARGIN.top + ")");
+
+  svg
+    .selectAll(".link")
+    .data(root.links())
+    .enter()
+    .append("line")
+    .attr("x1", function (d: any) {
+      return d.source.x;
+    })
+    .attr("y1", function (d: any) {
+      return d.source.y + 50;
```

Comment:
what are the numbers 50 and 150? Let's try not to use magic numbers and make them constants instead. Here and elsewhere in this function

Reply:
Done!